modules/auxiliary/gather/ie_uxss_cookie.rb

```diff
@@ -0,0 +1,148 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Auxiliary
+
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Microsoft Internet Explorer 10 and 11 Cross-domain Cookie Stealing",
+      'Description'    => %q{
+          This module exploits a universal cross-site scripting vulnerability found in Internet
+          Explorer 10 and 11. It will steal the cookie of a specific webiste (set by the TARGET_URI
+          datastore option). You will also most likely need to configure the MY_PUBLIC_IP
+          datastore option in order receive the cookie. If you and the victim are actually in the
+          same network, then you don't need to touch MY_PUBLIC_IP.
```

Comment:
"In the same network" If I'm on the Internet and the victim is on the Internet, are we "in the same network?" This is confusing.